src/plugins/dashboard/public/application/embeddable/grid/dashboard_grid.tsx

```diff
@@ -242,6 +242,17 @@ class DashboardGridUi extends React.Component<DashboardGridProps, State> {
       }
     });
 
+    const panelIds = [];
+    const startTime = new Date().getTime();
```

Comment:
Should we use [`performance.Now()`](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now) instead?